collections/hub/other_tests/bulk_transfers/bulk-invalid-timestamp.json

```diff
@@ -464,17 +568,63 @@
                 "id": 1,
                 "description": "Status code is 200",
                 "exec": [
-                  "expect(response.status).to.equal(200)"
+                  "expect(environment.participantsPositionResponse.code).to.equal(200)"
                 ]
               },
               {
                 "id": 2,
                 "description": "Payeefsp position after Payee ABORT should be same as position before prepare.",
                 "exec": [
-                  "expect(response.body[0].value).to.equal(+environment.payeefspPositionBeforePrepare)"
+                  "expect(environment.participantsPositionResponse.body[0].value).to.equal(+environment.payeefspPositionBeforePrepare)"
                 ]
               }
             ]
+          },
+          "scriptingEngine": "javascript",
+          "disabled": false,
+          "scripts": {
+            "postRequest": {
+              "exec": [
+                "const expectedStatusCode = 200;",
+                "environment.participantsPositionResponse = null",
+                "",
+                "// lets get retry config",
+                "const maxRetryCount = Number.parseFloat(environment.RETRY_MAX_ATTEMPTS) || 20",
+                "const maxRetryWait = Number.parseFloat(environment.RETRY_MAX_WAIT_IN_MS) || 250",
+                "",
+                "async function main() {",
+                "  let _response = response",
+                "",
+                "  for (let retryCount = 0 ; retryCount < maxRetryCount; retryCount++) {",
+                "    console.log(`retry-count=${retryCount+1}, max-count=${maxRetryCount}, max-wait=${maxRetryWait}`)",
+                "    ",
+                "    const isOk = _response.code === expectedStatusCode;",
+                "    ",
+                "    if (isOk) {",
+                "      environment.participantsPositionResponse = _response",
```

Comment:
If you don't need to access these variables in other requests, you can use requestVariables.